### PR TITLE
Add support for Animal Measurements (herd monitoring systems) #465

### PR DESCRIPTION
**Pull request by NedapLivestockManagement**
Create

### CODE REVIEW

File: icarAnimalMetricsCollection.json
Comment (mtech-ashelepaev) [inline icarAnimalMetricsCollection.json:1]: Is this a duplicate of the same file from the "collections" folder?

File: icarAnimalMetricsResource.json
Comment (mtech-ashelepaev) [inline icarAnimalMetricsResource.json:1]: Is this a duplicate of the same file from the "resources" folder?

File: enums/uncefactDurationUnitType.json
Comment (mtech-ashelepaev) [inline enums/uncefactDurationUnitType.json:1]: This new enum can be also referenced in icarMilkDurationType and icarFeedDurationType types.

File: types/icarAnimalMetricType.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  "properties": {
+    "timestamp": {
+      "$ref": "../types/icarDateTimeType.json",
+      "description": "The start date and time of the metric in UTC."
+    },
+    "isComplete": {
+      "type": "boolean",
+      "description": "Flag that indicates if the metric is complete. For daily metrics this flag is complete if the value is based on a full day of metrics. If not all metrics have been received the value will be false."
+    },
+    "type": {
```
Comment (mtech-ashelepaev) [inline types/icarAnimalMetricType.json:15]: "type" is a very generic word in many languages. Can we change it to "Period" or anything else, but "type"?

**Comment by cookeac** [comment]
Replaced by #486

**Pull request closed without merging**